app/views/admin/roles/_appointments_tab.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= render "govuk_publishing_components/components/button", {
+  text: "New appointment",
+  href: new_admin_role_role_appointment_path(role, make_current: true)
+} %>
+
+<%= render partial: "admin/role_appointments/list", locals: {role_appointments: role.role_appointments} %>
```

Comment:
i've held of saying this on other PRs but there's not longer a need to specify you're rendering a partial and pass locals in via a hash. This will likely be deprecated at some point so it'll be easier for us in the long run if we just use

```
<%= render "admin/role_appointments/list", role_appointments: role.role_appointments %>

```

if that's okay please